csi.proto

```diff
@@ -332,15 +330,8 @@ message ControllerUnpublishVolumeRequest {
   // The ID of the volume. This field is REQUIRED.
   string volume_id = 2;
 
-  // The ID of the node. This field is OPTIONAL. The CO SHALL set (or
-  // clear) this field to match the node ID returned by `GetNodeID`.
-  // `GetNodeID` is allowed to omit node ID from a successful response;
-  // in such cases the CO SHALL NOT specify this field.
-  //
-  // If `GetNodeID` does not omit node ID from a successful response,
-  // the CO MAY omit this field as well, indicating that it does not
-  // know which node the volume was previously used. The Plugin SHOULD
-  // return an Error if this is not supported.
+  // The ID of the node. This field is REQUIRED. The CO SHALL set this
+  // field to match the node ID returned by `GetNodeID`.
```

Comment:
We killed a use case here by making this REQUIRED: a CO may intentionally leave this field blank to indicate to the plugin that the volume should be unpublished from every node in the cluster. Is killing that use case really our intent here?

Reply:
Yeah. I think we deliberately made this field optional to capture the case where the CO just want to detach the volume from any node.  I still believe this is useful in some reconciliation scenarios.

Reply:
Ack. I'll restore this then.